slycot/tests/test_ab08n.py

```diff
@@ -1,5 +1,5 @@
 # ===================================================
-# ag08bd tests
+# ab08nX tests
```

Comment:
There is also an AB08NX.f in the SLICOT sources. Should we then later have an ab08nX and an ab08nx test file?

Reply:
```suggestion
# ab08n* tests
```